dependent-service-downtime-and-ps-adaptor-behavior.md

```diff
@@ -0,0 +1,64 @@
+## Service Failures and Expected Behavior
+
+This guide outlines the expected behavior of the PS Adaptor when various dependent services are down. 
+The PS Adaptor works in conjunction with several services including the GP2GP Translator, MHS Adaptor, Facade, and others. 
+The guide explains the effects of outages and how the system behaves in each case.
+
+### System Overview
+The PS Adaptor initiates a conversation for transferring patient records, relying on several interconnected services:
+- GP2GP Translator: Sends patient records to the PS Adaptor via a queue.
+- Facade Application: Works with the PS Adaptor to manage transfer requests.
+- MHS Adaptor: Manages inbound and outbound queues, which are responsible for transferring patient records to the incumbent system.
+- Other Dependencies: Include PS Adaptor database, MHS inbound and outbound services, file storage, Redis cache, and MHS outbound SDS.
+
+### Service Outage Scenarios and Expected Behavior
```

Comment:
I find the word "expected" used here and below a bit confusing. What is the use of this word trying to achieve?

Reply:
"Expected" meaning that we expect the components to react or behave in a particular manner once a specific situation occurs...

Reply:
ok, I will reword the subtitle based on your suggestion